setup.py

```diff
@@ -1,13 +1,16 @@
 from setuptools import setup, find_packages
 setup(
     name="docserv",
-    version="0.1.4",
+    version="0.9.6",
     packages=find_packages(),
     scripts=['bin/docserv', 'bin/docserv-stitch'],
     install_requires=[],
-    package_data={
-        '': ['config/docserv.ini', 'config/config.d/sles.xml', 'systemd/docserv.service']
-    },
+    data_files=[
+        ('/etc/docserv/', ['config/docserv.ini']),
+        ('/etc/docserv/config.d/', ['config/config.d/sles.xml']),
+        ('/usr/lib/systemd/system/', ['systemd/docserv.service']),
+        ('/usr/share/docserv/templates/', ['templates/index.html'])
+    ],
```

Comment:
In [line 22](https://github.com/SUSEdoc/docserv/pull/6/files#diff-2eeaed663bd0d25b7e608891384b7298L22); I think we shouldn't support 3.3. This is quite old.